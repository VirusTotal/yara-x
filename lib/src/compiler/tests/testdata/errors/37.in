rule test {
  strings:
    $a = /abc[xyz/
  condition:
    $a
}