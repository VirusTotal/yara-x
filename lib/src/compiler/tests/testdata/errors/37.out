error: invalid regular expression
 --> line:3:14
  |
3 |     $a = /abc[xyz/
  |              ^ unclosed character class
  |